### ISSUE TITLE
Hint on where the configuration is contained

### ISSUE THREAD

**Issue by cRoCx** [misterunknown/ifm#181]
When you look for it, you can find out the configuration entries lie inside the `src/main.php` file. I think it would be worth mentioning this somewhere in the documentation (`README.md` or wiki).

At first glance, I was actually opening several other files, while I mistakenly expected to find a dedicated config file.

**Comment by misterunknown** [comment]
Hm, the [README.md](https://github.com/misterunknown/ifm#configuration) hints already at the existing [wiki page](https://github.com/misterunknown/ifm/wiki/Configuration). I'm not sure where to put it more noticable.

**Comment by cRoCx** [comment]
You are right. I have missed that. Probably because I am too much used to see configuration instructions right after installation instructions and then I have scrolled too far, after the keybindings came just in between. My bad.

I have made a small change to your configuration wiki page, now that I found it. If you don't like it, then feel free to roll it back.

Thank you btw. for this great work here! I have recommended this repo to a few friends.
